cmd/sysbox-fs/main.go

```diff
@@ -264,6 +264,9 @@ func main() {
 
 			logrus.SetOutput(f)
 			log.SetOutput(f)
+		} else {
+			logrus.SetOutput(os.Stdout)
```

Comment:
Same as for the sysbox-mgr PR: let's log to `os.stderr` instead.

Reply:
Done!